mbed/SPI.h

```diff
@@ -118,7 +118,8 @@ class SPI {
      * @param rx_length The length of RX buffer
      * @param callback  The event callback function
      * @param event     The logical OR of SPI events to modify. Look at spi hal header file for SPI events.
-     * @return Zero if the transfer has started, or -1 if SPI peripheral is busy
+     * @retval 0 if the transfer has started or has been queued
+     * @retval -1 if SPI peripheral is busy or the transfer could not be queued
```

Comment:
Why is it not true?
`queue_transfer` can return `-1`, either if the queue is full, or, when no queue is specified, the peripheral is busy.